lib/kuma/macros/deprecated-badge.js

```diff
@@ -0,0 +1,5 @@
+export default function DeprecatedBadge() {
+  return `<abbr class="icon icon-deprecated" title="Нерекомендовано. Не для використання на нових вебсайтах.">
+        <span class="visually-hidden">Нерекомендовано</span>
```

Comment:
Ймовірно, ти мав на увазі щось таке
```suggestion
        <span class="sr-only">Нерекомендовано</span>
```